lib/dune

```diff
@@ -0,0 +1,7 @@
+(library
+  (name        conduit)
```

Comment:
you can drop the `(name)` and bump the version to 1.1 (lang & opam). If that's your thing :)